crates/wasi-crypto/src/wiggle_interfaces/symmetric.rs

```diff
@@ -197,6 +197,15 @@ impl super::wasi_ephemeral_crypto_symmetric::WasiEphemeralCryptoSymmetric for Wa
         Ok((&*self).options_get_u64(symmetric_state_handle.into(), name_str)?)
     }
 
+    fn symmetric_state_clone(
+        &mut self,
+        symmetric_state_handle: guest_types::SymmetricState,
+    ) -> Result<guest_types::SymmetricState, guest_types::CryptoErrno> {
```

Comment:
```suggestion
    // Add your code here
    println!("Cloning symmetric state...");
```

Reply:
@Tyldah Errr... why? This is implemented by the next line.